### DISCUSSION TITLE
Performing several procedures

### DISCUSSION THREAD

**Discussion by AsafYagoda** [pymeasure/pymeasure#650] [Q&A] [answered]
Hello,
I would like to perform several procedures (one after another) and use the plotter for each of them.
Without the plotter I was able to run the procedures.
When adding the plotter, it crushed when it started the second procedure.
I tried to use wait_for_close for the plotter but it still crushed (after closing the plotter for the first procedure).
Is there a way to do what I'm trying to do?
Best Regards

**Comment by msmttchr** [comment, marked as answer]
If you're using the `Plotter` class, please be aware that is currently broken see https://pymeasure.readthedocs.io/en/latest/tutorial/graphical.html#using-the-plotter

Otherwise, it could help to have some info regarding the error you get.
